projects/demo/src/modules/components/editor/embed/html5/examples/1/index.less

```diff
@@ -0,0 +1,9 @@
+.html5-editor-example._cypress {
+    //noinspection CssInvalidPseudoSelector
+    video::-webkit-media-controls-current-time-display,
+    video::-webkit-media-controls-time-remaining-display,
+    audio::-webkit-media-controls-current-time-display,
+    audio::-webkit-media-controls-time-remaining-display {
```

Comment:
What do you think about putting all logic inside this file?
https://github.com/Tinkoff/taiga-ui/blob/a0093a010a9263f2880c5e10fafd3df7c76c657a/projects/demo/src/modules/app/cypress.less#L1

```less
._is-cypress-mode .ProseMirror {
    & video: ... {}
}
```

Reply:
I need to hide not only in .ProseMirror

Reply:
In addition, in other situations in the future, who knows, we can change it if necessary